ExecutionOrder.md

```diff
@@ -0,0 +1,65 @@
+# SOMAS2020 Simulation Execution Order

```

Comment:
move to `docs/` and name it `EXECUTION_ORDER.md`? also add a reference to it on the main readme.

Reply:
On it boss